compiler/rustc_hir/src/hir.rs

```diff
@@ -3482,13 +3482,16 @@ mod size_asserts {
     rustc_data_structures::static_assert_size!(super::Expr<'static>, 64);
     rustc_data_structures::static_assert_size!(super::Pat<'static>, 88);
     rustc_data_structures::static_assert_size!(super::QPath<'static>, 24);
+    #[cfg(not(bootstrap))]
     rustc_data_structures::static_assert_size!(super::Ty<'static>, 72);
```

Comment:
`#[cfg(not(bootstrap))]` seems unnecessary for unchanged assertions. Rebase error?

Reply:
Yes, fixed.